libbeat/docs/newdashboards.asciidoc

```diff
@@ -126,56 +148,11 @@ If Elasticsearch is running on a different host, then you can use the environmen
 $ ES_URL="http://192.168.3.206:9200" make import-dashboards
 -------------------------------
 
-The command has the following options:
+To see all the available options, check the options below or run:
 
 ["source","sh",subs="attributes"]
 ----------------------------------------------------------------------
-
```

Comment:
Agreed. I was planning to remove the output myself, but ruflin beat me to it. :-) 

Looks like the -h option is still shown in the example, so I think we're good here.